kratos/python_interface/__init__.py

```diff
@@ -12,6 +12,32 @@
 # import core library (Kratos.so)
 from Kratos import *
 
+def __ModuleInitDetail():
+    """
+    Configure the parallel environment.
+    This is done before initializing the kernel to ensure that MPI
+    and the parallel DataCommunicator are initialized when the Kernel is built.
+    It is defined as a function to avoid polluting the Kratos namespace with local variables.
+    """
+    import argparse
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--using-mpi", help="Configure Kratos for an MPI run.", action="store_true")
+    args = parser.parse_args()
+
+    if args.using_mpi:
+        try:
+            import KratosMultiphysics.mpi
```

Comment:
This adds `World` (the wrapper for `MPI_COMM_WORLD`) to the `ParallelEnvironment` singleton. The entire reason for adding this here (and `ParallelEnvironement` being a separate singleton and not part of the `Kernel`) is that this will allow us to have MPI-specific loggers ready for printing before initializing the Kernel (so no more hundreds of Kratos banners in MPI runs -- Note: the MPI Logger is not included in this PR.